api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,1207 @@
+swagger: '2.0'
+info:
+  title: Nakadi Event Bus API Definition
+  description: |
+    -----------
+    Definitions
+    -----------
+    Nakadi at its core aims at being a generic and content-agnostic event broker with a convenient API.
+    In doing this, Nakadi abstracts away, as much as possible, details of the backing messaging
+    infrastructure. The single currently supported messaging infrastructure is Kafka (Kinesis is planned
+    for the future).
+
+    In Nakadi every Event has an EventType, and a **stream** of Events is exposed for each registered
```

Comment:
For people who are not familiar with Kafka it's not clear the relation between EventType, **stream**. Might it be that one eventType goes to many streams? or several eventTypes reside in one **stream**?
What if I have one eventType which is fired once a day. Do we really need to create a whole **stream** (topic) for that?